tests/test_venue_request.py

```diff
@@ -903,7 +903,7 @@ def test_post_decision_stage(self, client, test_client, selenium, request_page,
                 'submission_revision_start_date': start_date.strftime('%Y/%m/%d'),
                 'submission_revision_deadline': due_date.strftime('%Y/%m/%d'),
                 'accepted_submissions_only': 'Enable revision for all submissions',
-                'submission_revision_remove_options': ['keywords', 'pdf']
+                'submission_revision_remove_options': ['keywords']
```

Comment:
Can you add to this test a Submission Revision Stage reference where the revision name is Supplementary_Material and it only has the supplementary_material field? then you can check if the invitations were correctly created.